README.md

```diff
@@ -585,6 +618,7 @@ but the command-line `curl` command works fine, this may be the problem.
 Command                                                     | Description
 ----------------------------------------------------------- | ------------------------------------------------------------
 `rustup default nightly`                                    | Set the default toolchain to the latest nightly
+`rustup set profile minimal`                                | Set the default "profile" (see [profiles](#profiles))
```

Comment:
Maybe

> Set the default "profile" *to minimal* (see [profiles](#profiles))

Though it is possibly better to use `default` profile here so that people who randomly copy-paste commands from internets do not switch to `minimal` by accident.

Reply:
I agree but at the same time, people who wants the `default` profile don't need this command at all because `default` is... the default profile :smile:

Reply:
On the other side... switching to `nightly` is also a not very safe thing, so `minimal` is ok here